src/connectionController.ts

```diff
@@ -327,6 +327,7 @@ export default class ConnectionController {
           productDocsLink: LINKS.extensionDocs(),
         }),
       logger: Object.assign(log, { mongoLogId }),
+      proxyOptions: {},
```

Comment:
yes. these used to come from compass-preferences, but now they are a required param (https://github.com/mongodb-js/compass/pull/6174/files). since I didn't find them being set in VSCode before, I am just passing an empty object. in Compass we do set `useEnvironmentalVariableProxies: true`, but I thought VSCode would go under 'no-proxy' (https://github.com/mongodb-js/compass/blob/35bf965a922e2f25452ff3b3347954aa1b2fa660/packages/compass-settings/src/components/settings/proxy-settings.tsx#L67). on another thought, would be good to get that confirmed. what do you think @Anemy?

Reply:
`no-proxy` lgtm!